docs/admin/registration.rst

```diff
@@ -0,0 +1,41 @@
+Artemis supports user registration based on the Jhipster template.
+User registration has to be enabled in one ``application-*.yml`` file and can be customized.
+
+Example:
+
+.. code:: yaml
+
+    artemis:
+        user-management:
+            use-external: false
+            registration:
+                enabled: true
+                allowed-email-pattern: ([a-zA-Z0-9_\-\.\+]+)@(tum.de)|(in.tum.de)|(mytum.de)
```

Comment:
Actually, there is another flaw as it seems:
```suggestion
                allowed-email-pattern: ([a-zA-Z0-9_\-\.\+]+)@(tum\.de|in\.tum\.de|mytum\.de)
```

Reply:
I am not a regex expert 😅

Reply:
I think I found the 2nd error already, but only replaced it my local (not committed) application.yml file
Thanks for noticing it 👍

Reply:
fixed